packages/terra-worklist-data-grid/tests/jest/Row.test.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+/* eslint-disable-next-line import/no-extraneous-dependencies */
+import { shallowWithIntl } from 'terra-enzyme-intl';
+import Row from '../../src/subcomponents/Row';
+import Cell from '../../src/subcomponents/Cell';
+
+// Source data for tests
+const dataFile = {
+  cols: [
+    { id: 'Column-0', displayName: ' Vitals' },
+    { id: 'Column-1', displayName: 'March 16' },
+    { id: 'Column-2', displayName: 'March 17', isSelectable: false },
+  ],
+  rows: [
+    {
+      id: '1',
+      cells: [
+        { content: 'Heart Rate Monitored (bpm)' },
+        { content: '' },
+        { content: '66', isMasked: true },
+      ],
+    },
+    {
+      id: '2',
+      cells: [
+        { content: 'Temperature Oral (degC)' },
+        { content: '36.7' },
+        { content: '36.9', isMasked: true },
+      ],
+    },
+    {
+      id: '3',
+      cells: [
+        { content: 'Cardiac Index (L/min/m2)' },
+        { content: '2.25' },
+        { content: '2.28', isMasked: false },
+      ],
+    },
+  ],
+};
+
+describe('Row', () => {
+  it('creates a row with the correct number of cells', () => {
+    const rowData = dataFile.rows[0];
+
+    const wrapper = shallowWithIntl(
+      <Row
+        rowIndex={99}
+        key={12}
+        id={rowData.id}
+        height="25"
+        cells={rowData.cells}
+        hasRowSelection={dataFile.rows[0].hasSelectableRows}
+        displayedColumns={dataFile.cols}
+        rowHeaderIndex={0}
+        onCellSelect={jest.fn}
+        onRowSelect={jest.fn}
+        tabStopColumnIndex={1}
+        selectedCellColumnId="Column-1"
+      />,
+    );
+
+    const tableRow = wrapper.find('tr.worklist-data-grid-row');
+    expect(tableRow).toHaveLength(1);
+    expect(tableRow.find(Cell)).toHaveLength(3);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('verifies the cell is created with the right props', () => {
```

Comment:
This seems like it's unit testing the cell rather than the row, is this really needed as a row unit test?

Reply:
It is. Since the Row builds the cells using data given to it, we are verifying that the Row is building the correct number of cells with the correct props.